src/components/CertificateViewer.tsx

```diff
@@ -34,15 +35,21 @@ export const CertificateViewer = ({
   handleSendCertificate,
 }: CertificateViewerProps) => {
   const tokenRegistryAddress = getTokenRegistryAddress(document);
-  const [templates, setTemplates] = useState([]);
+  const [templates, setTemplates] = useState<TemplateProps[]>([]);
   const [selectedTemplate, setSelectedTemplate] = useState("");
   const originalData = getData(document);
   const attachments = originalData?.attachments;
   const hasAttachments = attachments && attachments.length > 0;
 
   const updateTemplates = useCallback((templates) => {
-    setTemplates(templates);
-    setSelectedTemplate(templates[0].id);
+    // filter all templates that are renderable currently
+    const templatesModified = templates.filter((item: { type: string }) => {
+      return item.type === "custom-template" || item.type === "application/pdf";
```

Comment:
just to be sure, that means that renderer before 2.3.0 will not display anything (no type)

Reply:
as discussed, will cater to renderer before 2.3.0. made the necessary edits